packages/block-library/src/index.js

```
@@ -220,13 +230,29 @@ export const __experimentalGetCoreBlocks = () => [
 	commentsPaginationPrevious,
 	postComments,
 	postCommentsForm,
+  tableOfContents,
```

CI will complain 😓 

```suggestion
	tableOfContents,
```

---

Thanks! I resolved conflicts using Github UI and it silently used the wrong whitespace characters :D